dwitter/static/js/scrolling.js

```diff
@@ -16,6 +17,18 @@ window.onload = function() {
     registerStatsClickListeners(dweet);
   });
 
+  [].forEach.call(recordButtons, function(recordButton) {
```

Comment:
This only attaches the event to the first page of dweets, and is not called when new dweets are loaded with ajax when you scroll to the bottom. (Yeah, I know it should be done differently :P )

Reply:
Take a look at how stiaje did it for the remix link pull request

Reply:
Here's the magic: https://github.com/lionleaf/dwitter/pull/313/files#diff-166af2ad95d6cae2973ae18decaa1016R37

This asks jQuery to listen for _all_ click events on the body, but it filters them by the selector, so it only actually calls the handler function if the click was made on or below a `.share-button` element.

Disadvantages: Any click anywhere will have to be checked by jQuery. (Slightly more CPU)
Advantages: We only have to add one event listener to handle all the record buttons, even buttons added in the future. (That uses less memory than a separate event handler for each individual record button)